content/blog/2020/07/2020-07-23-windows-services-update.adoc

```diff
@@ -0,0 +1,165 @@
+* Fix the default domain name in the `serviceaccount` parameter (jira:JENKINS-12660[])
+* Fix archiving of old logs in the `roll-by-size-time` mode.
+
+As you may see, there are many improvements available with this version,
+and we hope that it will make Windows service installation even more reliable.
+Some of the changes in WinSW also replaced old workarounds in the Jenkins core,
+making the code more maintainable.
+
+=== Use-cases affected by .NET Framework 2.0 support removal
+
+If you use .NET Framework 2.0 to run the Jenkins Windows services,
+the following use-cases are likely to be affected:
+
+* Installing the Jenkins server as a Windows service from web UI.
+  The official MSI Installer supports .NET Framework 2.0 for the moment, but it will be changed in future versions.
+* Installing agents as Windows services from their GUI.
+  This feature is provided by in link:https://github.com/jenkinsci/windows-slave-installer-module[Windows Agent Installer Module] from the Jenkins core.
+* Installing agents over Windows Management Instrumentation (WMI) via the plugin:windows-slaves[WMI Windows Agents] plugin
+* Auto-updating of Windows service wrappers on agents installed from GUI.
+
+=== Upgrade guidelines
+
+If all of your Jenkins server and agent instances already use .NET Framework 4.0 or above,
+there are no special upgrade steps required.
+Please enjoy the new features!
+
+If you run the Jenkins server as a Windows Service with .NET Framework 2.0,
+this instance will require an upgrade of .NET Framework to version 4.0 or above.
+We recommend running with .NET Framework 4.6.1 or above,
+because this .NET version provides many platform features by default
+(e.g. TLS 1.2 encryption and strong cryptography),
+and Windows Service Wrapper does not have to apply custom workarounds.
+
+If you want to continue running some of your agents with .NET Framework 2.0,
+the following extra upgrade steps are required:
+
+1. Disable auto-upgrade of Windows Service Wrapper on agents by setting the 
+  `-Dorg.jenkinsci.modules.windows_slave_installer.disableAutoUpdate=true` flag on the Jenkins server side.
+2. Upgrade agents with .NET Framework 4.0+ by downloading the recent Windows Service Wrapper 2.x
+  version from link:https://github.com/winsw/winsw/releases[WiNSW GitHub Releases]
+  and manually replacing the wrapper ".exe" files in the agent workspaces.
+
+== What's next?
+
+We plan to continue expanding the Windows support in Jenkins,
+including providing official Docker images for newer Windows versions.
+For example, there is already a link:https://github.com/jenkinsci/docker-agent/pull/145[pull request] which will introduce official agent images for Windows Server Core LTSC 2019 and 
+for Windows Server Core and Nano Server 1909.
+We are also interested to keep expanding test coverage for Windows platforms, 
+barring we have enough resources and CI Infrastructure contributors.
```

Comment:
I infer the need for resources and contributors as part of the opening part of the sentence.  Also it is not a problem if you choose to leave it unchanged.
```suggestion
We are also interested to keep expanding test coverage for Windows platforms.
```